src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaYFlowRepositoryTest.java

```diff
@@ -85,16 +89,31 @@ public void shouldCreateFlow() {
         createYFlow(Y_FLOW_ID_1, FLOW_ID_1, FLOW_ID_2);
         createTestFlow(FLOW_ID_3, switch1, PORT_3, VLAN_2, switch2, PORT_2, VLAN_1);
 
+        assertTrue(yFlowRepository.findYFlowId(FLOW_ID_1).isPresent());
```

Comment:
We discussed this point with @niksv and decided not to write unit tests for the update and patch. It is quite tricky to write it as a unit test and it would require a lot of effort. Instead, I added a unit test for the mapper where all the job for this issue is done.